lesson_12/less12.py

```diff
@@ -0,0 +1,55 @@
+from collections.abc import MutableSequence
+
+
+class CrayonsBox(MutableSequence):
+
+    def __init__(self, iterable):
+        self._crayons = list(iterable)
+
+    def __len__(self):
+        return len(self._crayons)
+
+    def __getitem__(self, index):
+        return self._crayons[index]
+
+    def __setitem__(self, key, value):
+        self._crayons[key] = value
+
+    def __delitem__(self, key):
+        del self._crayons[key]
+
+    def insert(self, index, item):
+        return self._crayons[index][item]
```

Comment:
Please add a test for this method also; are you sure this works?

Reply:
changed the return for ini

also: reuploaded, because of folder structure
do not know how to remove the initial commit

Reply:
thought I removed this initial commit with my last commit, but no. let me know how I can fix this. thanks.